azcredentials/credentials.go

```diff
@@ -3,6 +3,7 @@ package azcredentials
 const (
 	AzureAuthCurrentUserIdentity = "currentuser"
 	AzureAuthManagedIdentity     = "msi"
+	AzureAuthWorkloadIdentity    = "wi"
```

Comment:
Lets be more verbose here and change this to `workloadidentity` so it's very clear